Documentation/Docs-CSS/navbar.css

```diff
@@ -0,0 +1,62 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--primary);
+  width: 90vw;
+  margin: 1rem auto;
+  padding: 0.5rem 1.5rem;
+  border-radius: 5px;
+  box-shadow: 0px 0px 30px 0px var(--primary);
+  position: sticky;
+  top: 10px;
+}
+
+.nav-section {
+  display: flex;
+  align-items: center;
+}
+
+.logo-full {
+  display: none;
+  margin-left: 1rem;
+  cursor: pointer;
+}
+
+.logo {
+  margin-left: 1rem;
+  cursor: pointer;
+}
+
+.hamburger {
+  font-size: 1.5rem;
+  cursor: pointer;
```

Comment:
I see `cursor: pointer` being used multiple times. You could instead create a class fo that & use it wherever required.

Reply:
Will do it. Thanks for the suggestion!